stream-chat-android-client/src/main/java/io/getstream/chat/android/client/notifications/handler/NotificationConfig.kt

```diff
@@ -1,19 +1,15 @@
 package io.getstream.chat.android.client.notifications.handler
 
+import android.content.Context
 import io.getstream.chat.android.client.R
+import io.getstream.chat.android.client.models.Device
 
 public data class NotificationConfig(
     val notificationChannelId: Int = R.string.stream_chat_notification_channel_id,
     val notificationChannelName: Int = R.string.stream_chat_notification_channel_name,
     val smallIcon: Int = R.drawable.stream_ic_notification,
-    val firebaseMessageIdKey: String = "message_id",
-    val firebaseMessageTextKey: String = "message_text",
-    val firebaseChannelIdKey: String = "channel_id",
-    val firebaseChannelTypeKey: String = "channel_type",
-    val firebaseChannelNameKey: String = "channel_name",
     val errorCaseNotificationTitle: Int = R.string.stream_chat_notification_title,
     val errorCaseNotificationContent: Int = R.string.stream_chat_notification_content,
-    val useProvidedFirebaseInstance: Boolean = true,
```

Comment:
Let's mention that we are removing those properties because someone might be using them

Reply:
It is related with the previous comment, let me add a bit more info on the changelog about that